py-polars/src/interop/arrow/to_py.rs

```diff
@@ -49,3 +56,74 @@ pub(crate) fn to_py_rb(
 
     Ok(record.to_object(py))
 }
+
+/// Export a series to a C stream via a PyCapsule according to the Arrow PyCapsule Interface
+/// https://arrow.apache.org/docs/dev/format/CDataInterface/PyCapsuleInterface.html
+pub(crate) fn series_to_stream<'py>(
+    series: &'py Series,
+    py: Python<'py>,
+) -> PyResult<Bound<'py, PyCapsule>> {
+    let field = series.field().to_arrow(CompatLevel::oldest());
```

Comment:
I do think this should be `newest`, otherwise we trigger a copy whereas the consumer should decide if they want to cast to a datatype they can support.

Reply:
> > Why requested_schema is not used?
> 
> Does the protocol allow for this?

https://arrow.apache.org/docs/dev/format/CDataInterface/PyCapsuleInterface.html#schema-requests

Reply:
There's been discussion about this in https://github.com/apache/arrow/issues/39689. To be able to pass in a `requested_schema` argument, the consumer needs to know the schema of the producer's existing Arrow data. Only then can it know whether it needs to ask the producer to cast to a different type. 

I believe I summarized the consensus in https://github.com/apache/arrow/issues/39689#issuecomment-2245805836, but while waiting for confirmation, I think it would be best for us to leave `requested_schema` and schema negotiation to a follow up PR, if that's ok.